cmd/agent/main.go

```diff
@@ -87,6 +88,10 @@ func main() {
 
 	var qp client.LeveledQuoteProvider
 	vtpmAttest := vtpm.Attest
+	fetchAzureToken := vtpm.FetchAzureAttestation
```

Comment:
this function seems stateless so just import it from agent rather than adding it as a paramter agent.New()